### ISSUE TITLE
Presentation materials for instructors using OSX only

### ISSUE THREAD

**Issue by pllim** [astropy/astropy-workshop#76]
Presentation materials too Mac OSX-centric. For instance, while working on #74, I have no idea how to open the `Install.key` file on both Linux (RHEL 7) and Windows 10. Thus, I was unable to view/edit it.

In other sections, I see PDF but that is view-only, so I still cannot edit them as a potential instructor who does not own a Mac.

**Comment by pllim** [comment]
Why not Google Slide?

**Comment by bsipocz** [comment]
Yes, please migrate over to google slides. Even having a Mac doesn't mean that Keynote is available.

**Comment by eteq** [comment]
This is a good point.  I think maybe we should shift to ppt since that's basically "lowest common denominator" (i.e., Macs, Linux, and Windows can all open ppts).

> Why not Google Slide?

The problem here is that we then can't embed them in the github repo, which has historically been critical to ensure the notebooks and lectures are in sync with each other.  If there's some way to embed a *specific* version of s Google slide show I'd be all for that, though?

**Comment by bsipocz** [comment]
You can still do a pdf dump of the slides. One can easily name versions, I suspect it's possible to link to the named versions in addition of the pdf.
For a fact I don't have ppt on my laptop either. (surely I can install keynote, but just saying that I had the impression that the cloud is the future :trollface: )

**Comment by adrn** [comment]
I don't have powerpoint, so I guess Google slides is the way to go...

**Comment by nstarman** [comment]
What about [``beamer``](https://www.overleaf.com/learn/latex/Beamer)?

**Comment by nstarman** [comment]
Or perhaps even better, the tutorials can be written in [markdown with live code examples](https://rmarkdown.rstudio.com) and [auto-transpiled](https://bookdown.org/yihui/rmarkdown/beamer-presentation.html) into [beamer](https://www.overleaf.com/learn/latex/Beamer) and to PDF with a custom beamer theme.
